scripts/generate.v3.ts

```diff
@@ -35,6 +35,80 @@ const ethereumDocumentConfig = {
   timeout: 5 * 60 * 1000, // 5 min timeout for contract to execute
 };
 
+interface OutLocations {
+  revoked: string;
+  notRevoked: string;
+}
+interface GenerateRevocationStoreArgs {
+  logMessage: string;
+  document: v3.OpenAttestationDocument;
+  outLocations: OutLocations;
+}
+
+const baseDnsDidDocumentRevocationStore = {
+  ...baseDnsDidDocument,
+  openAttestationMetadata: {
+    ...baseDnsDidDocument.openAttestationMetadata,
+    proof: {
+      ...baseDnsDidDocument.openAttestationMetadata.proof,
+      revocation: {
+        type: v3.RevocationType.RevocationStore,
+        location: ethereumDocumentConfig.documentStore,
+      },
+    },
+  },
+};
+const baseDidDocumentRevocationStore = {
+  ...baseDidDocument,
+  openAttestationMetadata: {
+    ...baseDidDocument.openAttestationMetadata,
+    proof: {
+      ...baseDidDocument.openAttestationMetadata.proof,
+      revocation: {
+        type: v3.RevocationType.RevocationStore,
+        location: ethereumDocumentConfig.documentStore,
+      },
+    },
+  },
+};
+
+const generateRevocationStore = async ({ logMessage, document, outLocations }: GenerateRevocationStoreArgs) => {
+  info(`${logMessage}`);
```

Comment:
Wondering why you parameterized this. There is a coupling in between the two function in which the caller knows that the log message will be logged as is at the top.